scipy/stats/_hypotests.py

```diff
@@ -1014,11 +1014,15 @@ def boschloo_exact(table, alternative="two-sided", n=32):
     is a uniformly more powerful alternative to Fisher's exact test
     for 2x2 contingency tables.
 
+    Boschloo's exact test uses the p-value of Fisher's exact test as a
+    statistic, and Boschloo's p-value is the probability under the null
+    hypothesis of observing such an extreme value of this statistic.
```

Comment:
Moved from below because it is important enough to mention above and needs to be understood before the new comments about two-sided pvalues.